src/api/config/routes/webui.rb

```diff
@@ -348,7 +348,7 @@
     post 'request/set_bugowner_request' => :set_bugowner_request
     get 'request/:number/request_action/:id' => :request_action, as: 'request_action'
     get 'request/:number/request_action/:id/changes' => :request_action_changes, as: 'request_action_changes'
-    get 'request/:number/request_action/:request_action_id/inline_comment/:line' => :inline_comment, constraints: cons, as: 'request_inline_comment'
+    get 'request/:number/request_action/:request_action_id/inline_comment' => :inline_comment, constraints: cons, as: 'request_inline_comment'
```

Comment:
Hm I don't get why this is changing. Can you explain?

Reply:
Sure. An inline comment for a request action requires two parameters: a file and a number.

Before, we had them in a string with the form "diff_FILEINDEX_nLINENUMBER" and passed it to the `line` path parameter of that endpoint.

Now, we pass the splited parameters in different query parameters. This makes it easier to create links for this endpoint in our views, and also parse the file index and line number of that endpoint.